src/components/StringFormatter/_index.scss

```diff
@@ -0,0 +1,18 @@
+////
+/// String formatter.
+/// @group string-formatter
+/// @copyright IBM Security 2019
+////
+
+@import '../Component/mixins';
+
+@include component($name: string-formatter) {
+  display: inline-block;
+
+  &--truncate {
+    max-width: 80ch;
```

Comment:
I realize we are dealing with a character width here, but that `80` is still divisible by the mini unit 🤔 Could you use mini-units here for the value?